CI.hs

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env stack runhaskell --package extra --package optparse-applicative --
```

Comment:
You might want to specify a resolver here to avoid this breaking when versions change. It probably picks up the resolver from the `stack.yaml` but I’m not entirely sure I trust that especially if you invoke it from a different working directory.

Reply:
Added `--resolver lts-12.10` to match the stack.yaml file.